pkg/cmd/task/start.go

```diff
@@ -531,6 +537,39 @@ func (opt *startOptions) getInputs() error {
 		opt.Workspaces = append(opt.Workspaces, intOpts.Workspaces...)
 	}
 
+	if opt.UseParamDefaults && !opt.Last && opt.UseTaskRun == "" {
+		if err := opt.includeDefaultParams(opt.task); err != nil {
```

Comment:
This shouldn't be required, the controller (server-side) will do the work for us if we don't send those parameters.

Reply:
Yes exactly, and in that case, we need to remove this from pipeline start also.

Reply:
Yes, good catch. This was missed the first time around adding this with pipeline start.